com.unity.render-pipelines.core/Documentation~/exposing-preferences.md

```diff
@@ -0,0 +1,33 @@
+# Exposing preferences on the Core Render Pipeline Settings
+
+For exposing preferences on **Edit > Preferences > Core Render Pipeline**, you must specify classes that implement the interface `ICoreRenderPipelinePreferencesProvider`. Unity will automatically know what are the changes and will insert your preferences on the given page.
+
+For example:
+
+```
+    class MyPreference : ICoreRenderPipelinePreferencesProvider
+    {
+        class Styles
+        {
+            public static readonly GUIContent myBoolLabel = EditorGUIUtility.TrTextContent("MyBool", "This bool toggles my feature");
+        }
+
+        public List<string> keywords => new List<string>() {Styles.myBoolLabel.text};
+        public GUIContent header => EditorGUIUtility.TrTextContent("MyPreferenceSection", "Group of my preferences");
+
+        public static bool s_MyBoolPreference;
```

Comment:
Should you add info why static value are better here?

Reply:
It was simple example of the usage, I think a better approach should be a preferencekey or a ScriptableObject, but for example purposes I thought this is the simplest working case